testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -30,6 +31,19 @@ def retrieve_build_host_id
   visit("https://#{$server.full_hostname}/rhn/manager/cm/build")
 end
 
+Then(/^I wait until the image build is completed$/) do
+  step %(I wait at most 3300 seconds until event "Image Build suse_os_image scheduled by kiwikiwi" is completed)
+  step %(I wait at most 300 seconds until event "Image Inspect 1//suse_os_image:latest scheduled by kiwikiwi" is completed)
+end
+
+Then(/^I am on the new image page$/) do
+  step %(I navigate to "os-images/1/" page)
```

Comment:
We might want to research a different way to navigate here, based on the image name or the cmpute image name.